docs/sphinx/source/user_guide/roadmap.rst

```diff
@@ -0,0 +1,153 @@
+pvlib's :py:class:`~pvlib.pvsystem.PVSystem` and
+:py:class:`~pvlib.modelchain.ModelChain` classes connect the individual
+model functions together to provide coherent system simulations.
+However, not all of pvlib's models are accessible with these classes.
+For example:
+
+* Bifacial irradiance models
+* Internal, external, and horizon shading
+* Soiling loss models
+
+In addition to more comprehensive integration of pvlib's model functions,
+the scope of these high-level capabilities could be expanded:
+
+* Multi-inverter systems
+* Geographic and electrical coordinate systems, to enable more detailed modeling
+* "out-of-code" model specification (e.g. configuration files)
+
+
+Data I/O
+--------
+:py:mod:`pvlib.iotools` provides functions to import
+weather datasets, horizon profiles, component specifications,
+and other external data.  Expanding this capability by
+adding functions for additional data sources is of significant interest.
+:py:mod:`pvlib.iotools` would also benefit from further standardization
+(``map_variables``, etc) and more documentation.
+
+See :discuss:`1528` for a detailed plan for :py:mod:`pvlib.iotools`.
+
+
+Speed improvements & benchmarking
+---------------------------------
+The value of improved computation speed is growing over time
+as pvlib is increasingly used in commercial APIs, large-scale
+research studies, and high-resolution timeseries simulations.
+However, benchmarking and improving the runtime of pvlib's codebase
+has received only sporadic and unfocused attention so far.
+Establishing a comprehensive set of speed (and potentially memory)
+benchmarks would be a good place to start.  Identified bottlenecks
+should be reviewed and made more efficient if possible.  If not,
+then alternative models could be implemented for cases where
+speed is the priority.
+
+As accelerator tools like Numba continue to mature, they should be
+evaluated for potential use in pvlib.  Although fast runtime speed is
+desirable, it must be weighed against code maintainability and portability.
+
+For modeling topics where runtime can be significant (e.g. solar position),
+the relevant User's Guide documentation section could include speed
+comparisons.
+
+
+Documentation
+-------------
+pvlib's documentation is overdue for a revision, with a focus on
+strategic organization.  See :issue:`329` for a specific proposal.
+
+Within that framework, better "getting started" tutorials are needed.
+However, the scope should probably stick to pvlib usage.  PV modeling in
+general is better left to other projects like the
+`PVPMC website <https://pvpmc.sandia.gov/>`_ or the
+`conference tutorials <https://pvsc-python-tutorials.github.io/>`_.
+
+The gallery of examples could be significantly expanded as well.
+An aspirational goal could be for every public function to be used
+in at least one gallery page.
+
+Additionally, it would be nice to have a project website (pvlib.org?)
+with scope going beyond code documentation.
```

Comment:
How would this be different from the GitHub and ReadtheDocs site?

Reply:
I would be nice to have a website that unites all the pvlib community and projects (pvlib-python, pvanalytics, etc.)

And giving people the website https://pvlib-python.readthedocs.io is kinda rough... pvlib.org would be an important step for pvlib in my opinion